debian/changelog

```diff
@@ -1,3 +1,66 @@
+ubuntu-advantage-tools (32) noble; urgency=medium
+
+  * d/postinst: ensure migrations happen in correct package postinst (GH: #2982)
+  * d/apparmor:introduce new ubuntu_pro_esm_cache apparmor policy
+  * New upstream release 32 (LP: #2060732)
+    - api:
+      + u.pro.attach.token.full_token_attach.v1: add support for attach
+        with token
+      + u.pro.services.disable.v1: add support for disable operation
+      + u.pro.services.enable.v1: add support for enable operation
+      + u.pro.detach.v1: add support for detach operation
+      + u.pro.status.is_attached.v1: add extra fields to API response
```

Comment:
Is adding more fields to an existing API allowed/fine? I suppose it is (just removing would be bad), but wanted to double check. (I found it easier to add the review comment here in the d/changelog, but obviously the changelog entry is not my point :)

Reply:
I'm not sure if we've officially documented it before, but we have considered adding new fields to be backwards-compatible and we've done this before. Changing fields (name or type) or removing fields would both require a new version.